app/views/affiliates/index.html.erb

```diff
@@ -1,6 +1,13 @@
 <p id="notice"><%= notice %></p>
 
-<h1>Affiliates</h1>
+<div class="row index-header">
+  <div class="col-md-8 title">
+    <span>Affiliates</span>
```

Comment:
That is not what it looks like after my update. I think you may have done something wrong.